packages/lesswrong/lib/collections/users/custom_fields.js

```diff
@@ -79,6 +79,33 @@ const karmaChangeSettingsType = new SimpleSchema({
   }
 })
 
+const partiallyReadSequenceItem = new SimpleSchema({
```

Comment:
Seems pretty important to me to have a `lastRead` date or something like that, and to capture that data early, since that will enable us to remove the recommendations after some time-horizon has passed.

Reply:
We have that data, just not denormalized here--it's just the read-date of `lastReadPostId`, which is in `lwevents` and `readstatuses`.